bootstrap_cfn/fab_tasks.py

```diff
@@ -275,13 +275,18 @@ def enter_maintenance(maintenance_ip, dry_run=False):
     r53_conn = get_connection(R53)
 
     cached_zone_ids = {}
-    for elb in cfn_config.data['elb']:
-        if not apply_maintenance_criteria(elb):
-            continue
-        zone_id = get_cached_zone_id(r53_conn, cached_zone_ids, elb['hosted_zone'])
-        zone_name = get_zone_name()
-        print green("Attempting to update: \"{0}\":\"{1}\"".format(elb, maintenance_ip))
-        r53_conn.update_dns_record(zone_name, zone_id, elb, 'A', maintenance_ip, dry_run=dry_run)
+    try:
+        for elb in cfn_config.data['elb']:
+            if not apply_maintenance_criteria(elb):
+                continue
+            zone_id = get_cached_zone_id(r53_conn, cached_zone_ids, elb['hosted_zone'])
+            zone_name = get_zone_name()
+            print green("Attempting to update: \"{0}\":\"{1}\"".format(elb['name'], maintenance_ip))
+            r53_conn.update_dns_record(zone_name, zone_id, elb['name'], 'A', maintenance_ip, dry_run=dry_run)
+    except KeyError:
```

Comment:
here I only added try/except to catch KeyError and boto Error. the rest of code remains the same.